app/static/js/displayFilesMacro.js

```diff
@@ -21,10 +21,18 @@ $(document).ready(function(){
         });
 
     $('.attachmentCheck').change(function() {
+        // Store the current checkbox state
+        var isChecked = $(this).is(':checked');
```

Comment:
It's a good idea to keep variable creation as close as possible to where it is used. In this case, that means right before the `if`